lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -48,19 +47,31 @@ final class AddComicViewController: UIViewController, AlertDisplaying {
 
     private var finishButton = UIBarButtonItem()
     private var activeField: UITextField?
-    private var addComicViewControllerTextFields: [UITextField]?
+    private var comicFormViewControllerTextFields: [UITextField]?
     private var overlayButton = UIButton()
 
     // MARK: - Constants
 
     private let cameraViewController = CameraViewController()
+    private var comicFormViewModel: ComicFormViewModel?
```

Comment:
This can be a `let` and it doesn't need to be optional since you set it in the init.